src/i18n/locale/en.json

```
@@ -820,6 +820,7 @@
   "components.Settings.SettingsJobsCache.runnow": "Run Now",
   "components.Settings.SettingsJobsCache.sonarr-scan": "Sonarr Scan",
   "components.Settings.SettingsJobsCache.unknownJob": "Unknown Job",
+  "components.Settings.SettingsJobsCache.useravatars": "User Avatars",
```

```suggestion
  "components.Settings.SettingsJobsCache.useravatars": "User Avatars",
```
This doesn't seems good to me, maybe "User(s) Avatars or "User Avatar"